terraform/main.tf

```diff
@@ -70,7 +70,7 @@ module "cluster" {
       auto-upgrade = false
       disk-size    = "50"
       disk-type    = "pd-ssd"
-      image        = "COS"
+      image        = "UBUNTU"
```

Comment:
🤔 is this intended?

Reply:
(as part of "address terraform 0.12.16 deprecation warnings")

Reply:
looks like this PR built on top of #82

Reply:
Yup shoulda rebased